maro/simulator/scenarios/supply_chain/business_engine.py

```diff
@@ -114,8 +114,10 @@ def _build_world(self) -> None:
         self.world.build(conf, self.calc_max_snapshots(), self._max_tick)
 
     def _on_action_received(self, event: CascadeEvent) -> None:
-        actions: List[SupplyChainAction] = event.payload
+        assert isinstance(event.payload, list)
```

Comment:
EventBuffer will covert the given actions to list

Reply:
So maybe we should change the type hint of `actions` in event buffer from `object` to list.